src/App.tsx

```diff
@@ -1,25 +1,15 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { Route, BrowserRouter, Routes } from 'react-router-dom';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path='/'     element = {<div>Start page</div>}/> 
+        <Route path='/test' element = {<div>test</div>}/>
```

Comment:
Is it going to be part of the app?

Reply:
Its just some test routes. I have added it to check router functionality. Its going to be replaced by real components as Login page or Registration page.